### PR TITLE
Move server-sdk files into packages subdirectory

### PR DESCRIPTION
**Pull request by lukasIO**
In order to keep the commit history of the node-repo, we'll have to temporarily allow regular merge-commits instead of squashing them.

### CODE REVIEW

**Comment by davidzhao** [comment]
I've enabled merge commits here

File: .changeset/config.json [hunk 1]
```diff
@@ -1,11 +1,15 @@
 {
-  "$schema": "https://unpkg.com/@changesets/config@2.0.0/schema.json",
-  "changelog": ["@livekit/changesets-changelog-github", { "repo": "livekit/server-sdk-js" }],
+  "$schema": "https://unpkg.com/@changesets/config@2.2.0/schema.json",
+  "changelog": [
+    "@livekit/changesets-changelog-github",
+    {
+      "repo": "livekit/node-sdks-temp"
```
Comment (theomonnom) [inline .changeset/config.json:6]: I guess this should be changed?

**Review by theomonnom — APPROVED** [review]
🚀🚀🚀
let's rename the repo to livekit/node-sdks?

**Comment by lukasIO** [comment]
@nbsp I also copied over most of your README proposal for the node real-time SDK, I hope that's alright with you!

**Review by nbsp — CHANGES_REQUESTED** [review]
this line no longer applies, we removed it in the new PR (which you should also cherry-pick over)

**Comment by lukasIO** [comment]
I don't see any line associated with your comment, maybe a GH bug? 
I'm assuming you mean the `dispose` call?

**Comment by nbsp** [comment]
yep

**Comment by nbsp** [comment]
also, weird nit, it feels odd that our two subfolders are `rtc-node` and `livekit-server-sdk`. i know they're the package names and one of them is under a namespace — but it isn't consistent to have `livekit-` on one but not the other (and the prefix is redundant as well, maybe)

**Comment by davidzhao** [comment]
can we rename `rtc-node` to `livekit-rtc` ?

File: COPYING
Comment (nbsp) [inline COPYING:1]: out of `/COPYING`, `/NOTICE`, and `/LICENSE`, two of these can go, and COPYING/NOTICE are the same file
i think keeping `COPYING` is the best for GitHub UX without clashing with REUSE

**Comment by lukasIO** [comment]
> can we rename rtc-node to livekit-rtc ?

sure, I'll make that change and merge the PR. 

We had users (and also Russ) confused in the past as to why repos/folders and the npm package name weren't aligned, so I thought that would be a good shout here. `livekit-rtc` doesn't make a whole lot of sense to me when the package is called `@livekit/rtc-node`.

File: COPYING
Comment (lukasIO) [reply COPYING:1]: can you elaborate on the advantages UX wise? 

Does it hurt to keep `LICENSE` around anyways? It seems like pretty much every bigger repo has that file?

Comment (nbsp) [reply COPYING:1]: so `NOTICE` isn't picked up by GitHub as being a license file at all, and doesn't give it any special UI
between `LICENSE` and `COPYING` it doesn't really matter per se but i think it's redundant to have the same file once for REUSE and once for GitHub to play nice

Comment (lukasIO) [reply COPYING:1]: got it, thanks.

**Comment by nbsp** [comment]
is GH Actions set up to release version packages for rtc-node from this repo as well or not yet?

**Comment by lukasIO** [comment]
yeah, it's set up to release all packages in scope

**Pull request merged**